.github/workflows/build.yml

```
@@ -46,6 +46,21 @@ jobs:
           path: tools
           key: ${{ runner.os }}-tools-${{ hashFiles('recipe.cake') }}
 
+      - name: Setup netcoreapp3.1
+        uses: actions/setup-dotnet@v1.9.0
+        with:
+          dotnet-version: "3.1.413"
+
+      - name: Setup net5.0
+        uses: actions/setup-dotnet@v1.9.0
+        with:
+          dotnet-version: "5.0.403"
+
+      - name: Setup net6.0
+        uses: actions/setup-dotnet@v1.9.0
+        with:
+          dotnet-version: "6.0.100"
+
```

The newest version of `actions/setup-dotnet` can do:

```suggestion
      - name: Setup netcoreapp3.1
        uses: actions/setup-dotnet@v1.9.0
        with:
          dotnet-version: |
            3.1.413
            5.0.403
            6.0.100

```